tests/Twilio/Integration/Messaging/V1/DeactivationsTest.php

```diff
@@ -32,7 +32,7 @@ public function testFetchRequest(): void {
 
     public function testFetchResponse(): void {
         $this->holodeck->mock(new Response(
-            200,
+            307,
```

Comment:
This is a generated file (see top of file) which will be overwritten on the next release.

Reply:
Thanks.  I didn't see that.  If there some other place I should update?

Reply:
Check `VersionTest.php`

Reply:
Thanks.  I thought I was looking for something to test the deactivations API specifically.  I think I have it updated as you'd like now.